contracts/CellarPoolShare.sol

```diff
@@ -923,12 +923,10 @@ contract CellarPoolShare is ICellarPoolShare, BlockLock {
             uint256 outAmount0,
             uint256 outAmount1,
             uint128 liquiditySum,
-            uint256 fee0,
-            uint256 fee1
+            UintPair memory collectedFee
```

Comment:
We should use fee0, fee1 instead of collected fee. Because fee0 and fee1 include management fee and performance fee. not same as collected Fee.